src/sonic-yang-models/yang-models/sonic-feature.yang

```diff
@@ -0,0 +1,71 @@
+module sonic-feature{
+
+    yang-version 1.1;
+
+    namespace "http://github.com/Azure/sonic-feature";
+    prefix port;
+
+    import sonic-types {
+        prefix stypes;
+    }
+
+    description "Feature Table yang Module for SONiC OS";
+
+    container sonic-feature {
+
+        container FEATURE {
+
+            description "feature table in config_db.json";
+
+            list FEATURE_LIST {
+
+                key "name";
+
+                leaf name {
+                    description "feature name in Feature table";
+                    type string {
+                        length 1..32;
```

Comment:
Any reason for restricting the len to 32?

Reply:
I wanted to put a max length for the feature name. Thirty-two would be enough to cover all existing features. If we need a bigger string, we can modify in the future.